samples/dashboards/demo/axes-sync/demo.html

```diff
@@ -4,28 +4,31 @@
 <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
 
 <div id="container"></div>
-
+<p class="highcharts-description">
+  This demo shows how related charts can be synchronized. Hover over one
```

Comment:
Please also describe the data source and other syncing ways especially how to sync extremes between different components.